api/swagger/swagger-v1.yaml

```diff
@@ -3354,6 +3354,22 @@ paths:
           - all
           - public
           - unlisted
+      - name: gate_condition
+        in: query
+        description: Filter tracks by gate condition type. Multiple values can be provided to filter by multiple gate types (OR logic)
+        style: form
+        explode: true
+        schema:
+          type: array
+          items:
+            type: string
+            enum:
+            - ungated
```

Comment:
why do we need ungated? i guess not a problem just curious

Reply:
the idea is to be like "all usdc_gated and ungated". prob not a serious use-case though